hole/star-wars-gpt.go

```diff
@@ -113,6 +117,8 @@ var _ = answerFunc("star-wars-gpt", func() []Answer {
 	tests[10] = test{staticTest6, staticResult6}
 	tests[11] = test{staticTest7, staticResult7}
 	tests[12] = test{staticTest8, staticResult8}
+	tests[13] = test{staticTest9, staticResult9}
+	tests[14] = test{staticTest10, staticResult10}
```

Comment:
This can't have been run as it immediately panics with:
```
panic: runtime error: index out of range [13] with length 13
```

It's trivial to fix this by bumping the declared slice size but that also means we're changing the argc of this hole. #2128 also changed the argc but that was before the hole was stable and shipped.